scripts/create-kubestellar-demo-env.sh

```diff
@@ -70,16 +78,27 @@ context_clean_up() {
     done <<< "$output"
 }
 
+# Run context cleanup
 context_clean_up
 echo "Context space clean up completed"
 
+# Install KubeStellar core
 echo -e "\nStarting the process to install KubeStellar core: kind-kubeflex..."
-export KUBESTELLAR_VERSION=0.23.1
-
+# Check if KUBESTELLAR_VERSION is already set, otherwise use default
+if [ -z "${KUBESTELLAR_VERSION}" ]; then
```

Comment:
Each release contains a copy of this script, and the user is already being directed to use a copy of this script from a particular release. There is no need for the release to be a variable inside the script. It should be unconditionally fixed to the particular release that it appears in. Since these lines will next appear in release 0.25.0, this script should unconditionally set the kubestellar release string to "0.25.0".